src/index.html

```diff
@@ -8,6 +8,45 @@
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img
+        class="card_img"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h1 class="card_description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
```

Comment:
Consider will be better if you use tag h2 here
```suggestion
      <h2 class="card_description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```

Reply:
why is it better to use h2?